.github/workflows/android_deploy_production.yml

```diff
@@ -57,10 +57,15 @@ jobs:
       - name: Build Android apk
         run: flutter build apk --flavor production --release --build-number $GITHUB_RUN_NUMBER
 
+      - name: Get PR information
+        uses: 8BitJonny/gh-get-current-pr@2.2.0
+        id: PR
```

Comment:
As discussed [here](https://nimble-co.slack.com/archives/C01NKCFNE6R/p1699003402236439?thread_ts=1698896523.269649&cid=C01NKCFNE6R), we can use a super powerful action called https://github.com/marketplace/actions/release-changelog-builder. We can apply this action to generate release notes for merged PRs on the `main` branch, and with some updates, we can even use it to generate the latest merged PR info on the `develop` branch, too 🤓 

To generate release notes for merged PRs on the `main` branch, we need to determine a range of `fromTag` to `toTag` (both tag name or commit sha supported):
- fromTag: it uses the last tag if skipped.
- toTag: should be the HEAD commit.

```suggestion
      - name: Find HEAD commit
        id: head
        run: echo "sha=$(git rev-parse HEAD)" >> $GITHUB_OUTPUT

      - name: Build changelog on "main"
        id: changelog
        uses: mikepenz/release-changelog-builder-action@v4
        with:
          configuration: ".github/workflows/changelog-config.json"
          # Listing PRs from the last tag to the HEAD commit
          toTag: ${{ steps.head.outputs.sha }}
          token: ${{ secrets.GITHUB_TOKEN }}
```

Tested successfully here https://github.com/nimblehq/flutter-templates/actions/runs/6747818444/job/18344811492

Test branch https://github.com/nimblehq/flutter-templates/compare/chore/198-test-merge-1...test-lucas?expand=1

```
## 🧹 Chores

- [Chore] Bump version to 1.13.0
   - PR: #277

## Others

- [IGNORE][TEST] Ignore please
   - PR: #[27](https://github.com/nimblehq/flutter-templates/actions/runs/6747818444/job/18344811492#step:5:28)8
- [TEST] Ignore please
   - PR: #279
```